drivers/hmis/app/graphql/types/forms/form_item.rb

```diff
@@ -48,21 +48,9 @@ class Forms::FormItem < Types::BaseObject
     # nested children
     field :item, ['Types::Forms::FormItem'], 'Nested items', null: true
 
-    # Most boolean attributes default to false.
-    [:required, :prefill, :hidden, :read_only, :repeats, :warn_if_empty].each do |bool_attr|
-      define_method(bool_attr) do
-        object[bool_attr.to_s] || false
-      end
-    end
-
-    # By default, disabled items are hidden
+    # By default, disabled items are hidden. Leaving here to match legacy behavior.
     def disabled_display
       object['disabled_display'] || 'HIDDEN'
     end
-
-    # By default, item is enabled if any rules match
-    def enable_behavior
-      object['enable_behavior'] || 'ANY'
-    end
```

Comment:
There is a corresponding frontend change in https://github.com/greenriver/hmis-frontend/pull/827